server/base/src/main/java/org/apache/accumulo/server/conf/store/impl/PropCacheCaffeineImpl.java

```diff
@@ -130,13 +135,18 @@ public Builder(final ZooPropLoader zooPropLoader, final PropStoreMetrics metrics
     }
 
     public PropCacheCaffeineImpl build() {
-      return new PropCacheCaffeineImpl(zooPropLoader, metrics, ticker);
+      return new PropCacheCaffeineImpl(zooPropLoader, metrics, ticker, runTasksInline);
     }
 
     public Builder withTicker(final Ticker ticker) {
       this.ticker = ticker;
       return this;
     }
+
+    public Builder forTests() {
```

Comment:
You could pass in the ticker here and remove the withTicker() method - the ticker is only used for testing and if the in-lineing is also only for test, the functionallity could just be located in one method.  forTesting() seems to express that better than withTicker(), but either way.

Reply:
I thought about this, but I noticed that `withTicker` is used outside of the tests, in the `ZooPropStore` constructor.

Reply:
Looking now, that constructor is only called from a test. I can look at adding this change. Are there other tests that can use `forTests()` and remove `sleep` calls?

Reply:
The constructor in ZooPropStore looks like it could call forTests(ticker) - or an overloaded constructor added with the ticker created.

Reply:
Maybe? - Some of them are used to force the current, test thread to yield to help ZooKeeper notifications having an opportunity to propagate to set-up the follow-on test conditions.